pkg/node/manager/manager.go

```diff
@@ -293,13 +338,26 @@ func (m *manager) updateSubnetIfUsingENIConfig(cachedNode node.Node, k8sNode *v1
 			cachedNode.UpdateCustomNetworkingSpecs(eniConfig.Spec.Subnet, eniConfig.Spec.SecurityGroups)
 			return nil
 		}
-		return fmt.Errorf("failed to find subnet in eniconfig spec %s", eniConfigName)
 	} else {
 		cachedNode.UpdateCustomNetworkingSpecs("", nil)
 	}
 	return nil
 }
 
+func (m *manager) GetEniConfigNameFromCNINode(node *v1.Node) (string, error) {
+	cniNode, err := m.wrapper.K8sAPI.GetCNINode(types.NamespacedName{Name: node.Name, Namespace: config.KubeDefaultNamespace})
+	if err != nil {
+		return "", err
+	}
+	for _, feature := range cniNode.Spec.Features {
+		if feature.Name == v1alpha1.CustomNetworking && feature.Value != "" {
+			return feature.Value, nil
+		}
+	}
+
+	return "", apierrors.NewNotFound(schema.GroupResource{}, "couldn't find custom networking eniconfig name defined by aws-node")
```

Comment:
schema.GroupResource{} seems wrong. In fact, I'm not sure we want to use this library, since it's built for detecting not found on API calls. Maybe better to use fmt.Errorf?

Reply:
Makes sense. Change to fmt.